docs/python_package.md

```diff
@@ -56,19 +56,19 @@ objects in a single image message at a time and publish detection results on a d
 ## [`image_recognition_service.py`](../ros/src/mas_perception_libs/image_recognition_service.py)
 
 ### `RecognizeImageService`
-A wrapper for a [`RecognizeImage`](../../mcr_perception_msgs/srv/RecognizeImage.srv) service which uses
+A wrapper for a [`RecognizeImage`](../../mas_perception_msgs/srv/RecognizeImage.srv) service which uses
```

Comment:
these links will probably be broken, so better to remove them and refer to the `mas_perception_msgs` repo up top.

Reply:
Agree! Please push the changes and merge when you're ready!